tests/integration/test_outcome.py

```diff
@@ -115,6 +115,48 @@ def test_outcomes_custom_topic(
     assert start <= event_emission <= end
 
 
+def test_outcomes_two_configs(
+    get_topic_name, processing_config, relay, mini_sentry, outcomes_consumer
+):
+    """
+    Tests routing outcomes to the billing and the default topic based on the outcome ID.
+    """
+    project_config = mini_sentry.add_basic_project_config(44)
```

Comment:
I suggest we use `42` as the project key for projects we'll be sending events to. It's the project we use in integration tests, see
https://github.com/getsentry/relay/blob/4c76af43c6c8513995b066c210c5b8af936bb679/tests/integration/fixtures/mini_sentry.py#L271
```suggestion
    project_config = mini_sentry.add_basic_project_config(42)
```

Reply:
what the difference ?

Reply:
The endpoints in integration tests have the project ids hardcoded and fail for the rest of projects, see
https://github.com/getsentry/relay/blob/f9973a682eb3779f878ba6470bbbe89edd5f8e92/tests/integration/fixtures/mini_sentry.py#L287-L290